src/Dangl.SchneidControl/Services/DataLoggingScheduler.cs

```diff
@@ -17,12 +22,38 @@ public Task StartAsync(CancellationToken cancellationToken)
             Task.Run(async () =>
             {
                 DateTime lastStart;
+                DateTime? lastEmailSent = null;
                 while (_isRunning)
                 {
                     lastStart = DateTime.UtcNow;
                     using var scope = _serviceProvider.CreateScope();
                     var dataLoggingService = scope.ServiceProvider.GetRequiredService<IDataLoggingService>();
-                    await dataLoggingService.ReadAndSaveValuesAsync();
+                    var valuesResult = await dataLoggingService.ReadAndSaveValuesAsync();
+                    if (valuesResult.BufferTemperatureTop?.Value > 50 && valuesResult.BufferTemperatureTop?.Unit == "°C" && valuesResult.CurrentHeatingPowerDraw?.Value != 0)
+                    {
+                        var schneidControlSettings = _configuration.Get<SchneidControlSettings>();
+                        if (schneidControlSettings?.EmailRecipients?.Count > 0)
+                        {
+                            if (lastEmailSent == null || ((DateTime.UtcNow - lastEmailSent.Value) > TimeSpan.FromHours(12)))
```

Comment:
The logic here isn't right I think.

When we detect a too low temperature, we should:
* Iterate `foreach` every recipient
* For every recipient, check the db if there's already an email of that type from the last 12 hours
* If no email, then send one (but save it to the database first, so for invalid email config we don't try to send every five minutes)